public/markdown/Guide.md

```
@@ -85,8 +88,10 @@ And to do that, put this into the `<metadata>` element (right before the `</meta
 > ![An image showing the "Credists" (Now renamed to "Written by:") section in the Spotify App with Spicy Lyrics installed, at the bottom of the lyrics tab](https://wdeliverystatic.global.ic.spicylyrics.org/wclient/a-cdn/lyrprep/f9754fc997c7548c8d00e86ea71634fb42a7970e.png)
 
 ## 7. Checks and Uploading
-#### Enable "Dev Mode" in the extension's settings and upload your TTML file by clicking on the "`<>`" button at the top (default to be on the top but it's a setting, so might be at the bottom for you) of the lyrics page, then click on "Upload" and select the file.
-#### If it has uploaded and parsed successfully, all the lines are in place, and there are no errors, then go to the "[create-a-ticket](https://discord.com/channels/1369992682214264993/1420470843414413403)" channel in the Spicy Lyrics Discord Server.
+#### Enable "Dev Mode" in the extension's settings and load your TTML file by clicking on the "`<>`" button at the top (default to be on the top but it's a setting, so might be at the bottom for you) of the lyrics page, then click on "Load" and select the file.
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)